docs/modules/third_party_rules_callbacks.md

```diff
@@ -265,6 +265,24 @@ server_.
 
 If multiple modules implement this callback, Synapse runs them all in order.
 
+### `on_threepid_unbind`
```

Comment:
Maybe just `unbind_threepid`, wdyt?

Reply:
`unbind_threepid` kind of convey a replacement to me.

`on_` is blurry to me between right after and right before. I feel like `on_` + doc is fine but you are more skilled than me in English so I could be wrong :) `before_unbind_threepid` perhaps ?

Reply:
Well it's more that empirically in programming `on_` usually means the callback reacts to something that has already happened. For example, if you consider JavaScript's event listeners, you're going to register a callback e.g. "on click", meaning we're reacting to the event that's sent once the action has happened. `before_unbind_threepid` feels a bit awkward to me but could work ultimately.

I still think `unbind_threepid` makes sense, in that when we call it we're asking the module to unbind the threepid. Plus one could argue that it could be correct to understand it as a replacement since it can act as one (since you can tell Synapse to not go any further). But I'm pretty bad at naming things so maybe there's a better name for it, so feel free to share ideas if you have some.